shipitscript/shipitscript/script.py

```diff
@@ -61,8 +62,8 @@ def create_new_release_action(context):
         # TODO quit early, mark task as green though
         pass
     log.info('Determining most recent shippable revision')
-    shippable_revision = ship_actions.get_shippable_revision(
-        repo, last_shipped_revision
+    should_build = ship_actions.should_ship_revision(
```

Comment:
we need to use the revision below. see how `shippable_revision` is used.

Reply:
done